src/mixins/commands_mixin.js

```diff
@@ -336,6 +340,22 @@ export const commands_mixin = {
       get() { return this.selected_site },
       set(value) { this.$store.commit('site_config/setActiveSite', value) }
     },
+    active_subpage: {
+      get() { return this.selected_subpage },
+      set(value) { this.$store.commit('site_config/setActiveSubpage', value) }
+    },
+    active_target_tab: {
+      get() { return this.selected_target_tab },
+      set(value) {this.$store.commit('site_config/setActiveTargetTab', value) }
+    },
+    active_image_tools_tab: {
+      get() { return this.selected_image_tools_tab },
+      set(value) {this.$store.commit('site_config/setActiveImageToolsTab', value) }
+    },
+    active_controls_tab: {
+      get() { return this.selected_controls_tab },
+      set(value) { this.$store.commit('site_config/setActiveControlsTab', value) }
+    },
```

Comment:
Since these don't relate to sending commands, better to have them outside of the commands mixin. I would lean towards just adding these to each file that uses them, since it's clearer when trying to figure out where those values are coming from.